sbt/src/sbt-test/sbt-stryker4s/test-1/README.md

```diff
@@ -2,6 +2,8 @@
 
 This is a test project for the Stryker4s SBT plugin.
 
-To run it, first run `sbt sbt-stryker4s2_12/publishLocal` in the root Stryker4s project.
+To run it, first
+run `sbt stryker4s-core2_12/publishLocal; stryker4s-api2_12/publishLocal; sbt-stryker4s-testrunner2_12/publishLocal` in
```

Comment:
Following the readme didn't work locally, so I peeked at what the CI did and copied that, which worked.

Reply:
I think this can also be replaed with the command `sbt publishPluginLocal` which does all that (I think at least for 2.13 projects, don't know if it works for 2.12 projects)